packages/legacy/core/App/screens/Home.tsx

```diff
@@ -130,81 +94,48 @@ const Home: React.FC<HomeProps> = ({ navigation }) => {
-          onCallToActionPressed={onCTAPressed}
-          onCallToActionLabel={t('Tour.UseAppGuides')}
-          onSecondCallToActionPressed={onDismissPressed}
-          onSecondCallToActionLabel={t('Tour.DoNotUseAppGuides')}
-          onDismissPressed={onDismissPressed}
-        />
-      )}
-      <View style={styles.rowContainer}>
-        <View>
-          {notifications?.length > 0 ? (
-            <AttachTourStep index={1} fill>
-              <Text style={[HomeTheme.notificationsHeader, styles.header]}>
-                {t('Home.Notifications')}
-                {notifications?.length ? ` (${notifications.length})` : ''}
-              </Text>
-            </AttachTourStep>
-          ) : (
-            <Text style={[HomeTheme.notificationsHeader, styles.header]}>
-              {t('Home.Notifications')}
-              {notifications?.length ? ` (${notifications.length})` : ''}
-            </Text>
-          )}
-          {notifications?.length > 1 ? (
-            <TouchableOpacity
-              style={styles.linkContainer}
-              activeOpacity={1}
-              onPress={() => navigation.navigate(Screens.Notifications)}
-            >
-              <Text style={styles.link}>{t('Home.SeeAll')}</Text>
-            </TouchableOpacity>
-          ) : null}
-        </View>
-      </View>
+    <>
       <FlatList
-        horizontal
-        showsHorizontalScrollIndicator={false}
+        style={{ marginBottom: 35 }}
+        showsVerticalScrollIndicator={false}
         scrollEnabled={notifications?.length > 0 ? true : false}
-        snapToOffsets={[
-          0,
-          ...Array(notifications?.length)
-            .fill(0)
-            .map((n: number, i: number) => i * (width - 2 * (offset - offsetPadding)))
-            .slice(1),
-        ]}
         decelerationRate="fast"
         ListEmptyComponent={() => (
-          <View style={{ marginHorizontal: offset, width: width - 2 * offset }}>
+          <View style={{ marginHorizontal: 25, marginVertical: 20 }}>
             <AttachTourStep index={1} fill>
               <View>
                 <NoNewUpdates />
               </View>
             </AttachTourStep>
           </View>
         )}
+        ListHeaderComponent={() => <HomeHeaderView />}
+        ListFooterComponent={() => <HomeFooterView />}
         data={notifications}
         keyExtractor={(item) => item.id}
-        renderItem={({ item, index }) => (
+        renderItem={({ item }) => (
           <View
             style={{
-              width: width - 2 * offset,
-              marginLeft: !index ? offset : offsetPadding,
-              marginRight: index === notifications?.length - 1 ? offset : offsetPadding,
+              marginHorizontal: 20,
+              marginVertical: 10,
             }}
           >
             {DisplayListItemType(item)}
           </View>
         )}
       />
-      <HomeContentView />
-    </ScrollView>
+      {showTourPopup && (
```

Comment:
Did you test that this pop-up still works? Might have to do a fresh install to see

Reply:
Afirmative.

![IMG_CFCECAD14B18-1](https://github.com/hyperledger/aries-mobile-agent-react-native/assets/390891/882271c2-bee5-40c3-a5d9-66a62780bfb7)